Makefile

```diff
@@ -162,27 +163,30 @@ cluster-templates: $(KUSTOMIZE) ## Generate cluster templates
 ## --------------------------------------
 GINKGO_FOCUS  ?=
 GINKGO_SKIP ?=
+GINKGO_TIMEOUT ?= 6h
 
 ifneq ($(strip $(GINKGO_SKIP)),)
-_SKIP_ARGS := $(foreach arg,$(strip $(GINKGO_SKIP)),-ginkgo.skip="$(arg)")
+_SKIP_ARGS := $(foreach arg,$(strip $(GINKGO_SKIP)),-skip="$(arg)")
 endif
+
 .PHONY: e2e-tests
 e2e-tests: CONTAINER_RUNTIME?=docker ## Env variable can override this default
 export CONTAINER_RUNTIME
-e2e-tests: e2e-substitutions cluster-templates ## This target should be called from scripts/ci-e2e.sh
+e2e-tests: $(GINKGO) e2e-substitutions cluster-templates ## This target should be called from scripts/ci-e2e.sh
 	for image in $(E2E_CONTAINERS); do \
 		$(CONTAINER_RUNTIME) pull $$image; \
 	done
-	cd test; \
-	time go test -v -timeout 24h --tags=e2e ./e2e/... -args \
-		--ginkgo.timeout=6h --ginkgo.v --ginkgo.trace --ginkgo.show-node-events --ginkgo.no-color=$(GINKGO_NOCOLOR) \
-		--ginkgo.junit-report="junit.e2e_suite.1.xml" \
-		--ginkgo.focus="$(GINKGO_FOCUS)" $(_SKIP_ARGS) \
+
+	$(GINKGO) --timeout=$(GINKGO_TIMEOUT) -v --trace --tags=e2e  \
+		--show-node-events --no-color=$(GINKGO_NOCOLOR) \
+		--junit-report="junit.e2e_suite.1.xml" \
+		--focus="$(GINKGO_FOCUS)" $(_SKIP_ARGS) $(ROOT_DIR)/$(TEST_DIR)/e2e/ -- \
```

Comment:
ROOT_DIR may contain spaces so this breaks - needs quotes

Reply:
Same with $GINKGO as its using $ROOT_DIR earlier

Reply:
Thanks fixed